### DISCUSSION TITLE
are vaccinations working or omicron less lethal? Is omicron the same as seasonal flu?

### DISCUSSION THREAD

**Discussion by djay** [djay/covidthailand#235] [Q&A]
Some people point to the lower death rate during omicron as being because omicron itself is less lethal. However in many countries immunity is now much higher due to previous infections and vaccinations.

Another statistic some people use is try to say covid is not something to worry about is that 46k deaths from flu in thailand per year (precovid). this is actually 46k deaths from influenza and pneumonia combined, not just influenza. The actual deaths attributed to influenza in thailand is around 2-3k per year.

**Comment by djay** [comment]
TLDR; Vaccination seems to be the main reason omicron is resulting in less deaths. Omicron is about as deadly as original strain if unvaccinated but about only twice as deadly as flu when vaccinated (across the whole population). As of April 2022 40% of the Thai population haven't had omicron yet and it's a lot of more contagious than the flu and vaccination rates for elderly are not that high so Omicron is a concern.

I'm not aware of studies specifically about Thailand but omicron is the same virus so there is no reason to think it would affect thailand differently. 

While omicron is known to have immune escape properties that isn't completly true. Vaccinations and natural immunity reduce both severe illness and spread just less than previous variants. 

These calculators are updated for omicron and show you the risk if unvaccinated vs vaccinated - [AU Risk Calc](https://corical.immunisationcoalition.org.au/)

<img width="1148" alt="image" src="https://user-images.githubusercontent.com/41700/164879255-4b20d04b-0f17-4963-b527-9543548080ac.png">


IHME collect all the research on this subject and summarise it into the following table.

![](https://www.healthdata.org/sites/default/files/files/Projects/COVID/2022/vaccine-efficacy-table_011022.jpg)

Another [study looking at omicron with boostered immunity with different omicron subvariants](https://yourlocalepidemiologist.substack.com/p/state-of-affairs-jan-31) found

![](https://cdn.substack.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2F69d476c8-dd3f-48b3-b137-fc7690609e5e_988x242.jpeg)


# UK IFR vs Flu
[FT 2022-03-10: Vaccines and Omicron mean Covid now less deadly than flu in England](https://www.ft.com/content/e26c93a0-90e7-4dec-a796-3e25e94bc59b)
![](
![image](https://www.ft.com/__origami/service/image/v2/images/raw/https%3A%2F%2Fd6c748xw2pzm8.cloudfront.net%2Fprod%2F3b449b50-a317-11ec-bf5a-bd778a859a9f-standard.png?dpr=2&fit=scale-down&quality=medium&source=next&width=700)


# Omicron without any natural or vaccination immunity

[Some studies point to omicron being only 25% less lethal than delta when you have no immunity at all](https://www.bloomberg.com/news/articles/2022-01-14/omicron-less-severe-even-for-unvaccinated-s-african-study-says)


[CDC data from 1st week jan 2022 showing chance in one million for dying of covid](https://yourlocalepidemiologist.substack.com/p/understanding-risk?s=r). The chance of dying of omicron during that week if you were were unvaccinated and over 65 was about the same as 1.5 years of taking heroin.

![](https://cdn.substack.com/image/fetch/w_1456,c_limit,f_auto,q_auto:good,fl_progressive:steep/https%3A%2F%2Fbucketeer-e05bbc84-baa3-437e-9518-adb32be77984.s3.amazonaws.com%2Fpublic%2Fimages%2Fa6fa6043-d510-4a72-a539-bfe475a4f250_811x287.png)

The difference between HK and NZ is more evidence on how much immunity lowers the death rate
![](https://pbs.twimg.com/media/FN0isMFXsAQbJH8?format=jpg&name=4096x4096)

# Data from thailand


While not conclusive this CFR plot for thailand shows as that as vaccines increase fatality rate goes down. CFR is not a perfect measure as its very much influenced by your detection rate which is influenced by your testing system. But note that the regions with the most vaccination (bangkok) are the ones with the most drop in CFR. This shows that omicron is not the main reason why deaths are lower.

![](https://practical-ritchie-cca141.netlify.app/outputs/cfr_age_3.png)
![](https://practical-ritchie-cca141.netlify.app/outputs/cfr_region_3.png)
![](https://practical-ritchie-cca141.netlify.app/outputs/cfr_ifr_3.png)
![](https://practical-ritchie-cca141.netlify.app/outputs/vac_region_2_3.png)

Note

> According to the National Communicable Disease Committee, endemic status can be given to provinces with less than 10,000 new cases per day, and where the fatality rate does not exceed 0.1%. Another stipulation is that more than 80% of those at-risk have received coronavirus vaccine doses. Dr Suthep said if no new coronavirus variant emerges, he is confident Covid-19 will be declared an endemic disease by July 1.
- [Bangkok Post: 2022-04-25: No endemic status before EOC's nod](https://www.bangkokpost.com/thailand/general/2299630/no-endemic-status-before-eocs-nod)
> 4.3 To reach endemic stage, the fatality rate for COVID-19 must not exceed 0.1 percent and more than 60 percent of population must have received their booster dose.
[CCSA Briefing 2022-04-28](https://www.facebook.com/richardbarrowthailand/posts/560759782076956)

# Influenza in Thailand

- 1,231 out of every 100,000 get seasonal flu in Thailand each year (or avg of 263 infections per day across the year)
- "less than 0.1% fatality rate" (all age groups) and 11.4% over 65.
There are other [interesting comparisons between Flu, Pneumonia and Covid](https://www.samitivejhospitals.com/article/detail/covid-19-influenza-pneumonia)

So while the likely IFR in Thailand could now be lower than the flu (due to immunity), due to the much larger number of infections (as of May 2022) the number of deaths is much larger.


There is a number spread around of 2019 having 44,549 deaths from Influenza and Pnuemonia. But Pnuemonia has many other causes other than flu and that number is combined, not purely those from Influenza.

**Comment by djay** [comment]
Good look at omicron data in USA about risk for vaccinated and unvaccinated. Doesn't include risk if never had any previous infection. 
https://yourlocalepidemiologist.substack.com/p/understanding-risk?s=r

**Comment by djay** [comment]
Probably best evidence is from HK https://www.ft.com/content/6e610cac-400b-4843-a07b-7d870e8635a3 esp compared to NZ

**Reply by djay** [reply]
Some good data in https://yourlocalepidemiologist.substack.com/p/another-mrna-booster-or-not?s=r
